src/mixins/behaviors.js

```diff
@@ -89,6 +89,15 @@ export default {
     _invoke(this._behaviors, 'destroy', ...args);
   },
 
+  // Remove a behavior
+  _removeBehavior(behavior) {
+    // Don't worry about the clean up if the view is destroyed
+    if(this.view._isDestroyed) {
```

Comment:
Yeah I dunno.. I much [prefer guards](http://there4.io/2015/06/10/refactoring-with-guard-clauses-php-javascript/) myself..  and it's not uncommon in Mn to do so.. however in every case it's just `{ return; }` which I changed this to.. though.. if people feel strongly it's not a big deal in this function.